cobc/cobc.c

```diff
@@ -9252,6 +9277,19 @@ main (int argc, char **argv)
 	finish_setup_compiler_env ();
 	finish_setup_internal_env ();
 
+	{
+		struct cb_text_list *l;
+		for (l = cb_copy_list; l; l=l->next){
+			char *name = cobc_strdup(l->text);
```

Comment:
Do we need to duplicate the l->text? Why not using it directly?
If needed, then please use a local `char name[COB_MINI_BUFF]` here, this also removes the new failing under valgrind's memcheck when there is an error (because the err_exit leaves before we free the data).

Reply:
The reason is that `cb_copy_find_file()` expects a non-const first argument (it modifies `/` into '\\` under Windows for example), while `cb_text_list` contains a `const char*` string.

I used the local array to remove the warning from valgrind.

Reply:
Thank you for the explanation, adding a note that the buffer name point to is modified in its function doc is likely good (its enough to do before the commit to svn).

The change is good, was there also a reason (other than the late hour) to not use `strcpy`?